core/src/main/java/org/springframework/security/authorization/ObservationAuthorizationManager.java

```diff
@@ -73,12 +80,16 @@ public AuthorizationDecision check(Supplier<Authentication> authentication, T ob
 	/**
 	 * Use the provided convention for reporting observation data
 	 * @param convention The provided convention
-	 *
 	 * @since 6.1
 	 */
 	public void setObservationConvention(ObservationConvention<AuthorizationObservationContext<?>> convention) {
 		Assert.notNull(convention, "The observation convention cannot be null");
 		this.convention = convention;
 	}
 
+	@Override
+	public void setMessageSource(final MessageSource messageSource) {
+		this.messages = new MessageSourceAccessor(messageSource);
+	}
```

Comment:
Please add javadoc with at a minimum `@since 6.2` to document when this method was introduced.